docs/monitoring/debugging_event_time.md

```diff
@@ -31,11 +31,7 @@ Flink's [event time]({{ site.baseurl }}/dev/event_time.html) and watermark suppo
 out-of-order events. However, it's harder to understand what exactly is going on because the progress of time
 is tracked within the system.
 
-There are plans (see [FLINK-3427](https://issues.apache.org/jira/browse/FLINK-3427)) to show the current low watermark
-for each operator in the Flink web interface.
-
-Until this feature is implemented the current low watermark for each task can be accessed through the 
-[metrics system]({{ site.baseurl }}/monitoring/metrics.html).
+Low watermarks of each task can be accessed from either Flink web interface or Flink [metrics system]({{ site.baseurl }}/monitoring/metrics.html).
```

Comment:
I would reword it to "can be access through the Flink web interface or metrics system".